helpers.sh

```diff
@@ -117,3 +118,40 @@ init_broken_repo () {
     echo""
     ls
 }
+
+init_repo_remote () {
+    mkdir -p conference_planning
+    cd conference_planning
+    git init
+    cp ../../../examples/schedule_day1 .
+
+    git add schedule_day1 && git commit -m "Add schedule_day1"
+
+    ed -s schedule_day1  <<< $'/program/\na\n09:00-11:00: Poster session\n.\nw\nq' > /dev/null
+    git add * && git commit -m "Add poster sessions in the morning"
+
+    ed -s schedule_day1  <<< $'/session/\na\n11:00-11:15: Coffee break\n.\nw\nq' > /dev/null
+    git add * && git commit -m "Add coffee breaks"
+
+    ed -s schedule_day1  <<< $'/break/\na\n11:15-12:15: Talk professor A.\n12:15-13:30: Lunch\n13:30-15:00: Workshop\n15:00-16:00: Talk professor B.\n.\nw\nq' > /dev/null
+    git add * && git commit -m "Add rest of daily program"
+
+    cd ..
+    git clone conference_planning conference_planning_remote
+    cd conference_planning_remote
+    git switch -c "updated_schedules"
+    git switch main
+
+    cd ../conference_planning
+    ls
+}
+
+commit_to_remote() {
```

Comment:
Here a more meaningfu name would make sense:

do_a_commit_to_remote_by_a_third_party or so

Reply:
changed to commit_to_remote_by_third_party